.vscode/settings.json

```diff
@@ -1,6 +1,9 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.addMissingImports": "explicit"
```

Comment:
fyi, this makes sure missing imports are added automatically on save, I kept finding myself to do this manually (or AI-powered)